doc/driver.mld

```diff
@@ -698,2121 +718,132 @@ html/odoc/odoc_html/Odoc_html/Html_fragment_json
-$ '../src/odoc/bin/main.exe' 'html-generate' 'oprint.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'opcodes.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'mtype.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'meta.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'matching.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'makedepend.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'maindriver.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'main_args.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'main.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'local_store.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'liveness.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'linscan.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'linearize.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'linear_format.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'lift_let_to_initialize_symbol.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'lift_constants.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'lift_code.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'lexer.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'invariant_params.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'interval.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'interf.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'int_replace_polymorphic_compare.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'inlining_transforms.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'inlining_decision_intf.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-g
```

Comment:
The driver is full of tuples, which I find quite difficult to read especially without good tooling in mld files.

I think using a record eases the pain: there is a place where the type is defined (and maybe, documented?), you don't have to look for places where the tuples are created/used to infer the type yourself.

Reply:
This one is about line 746 of the new file.